apps/examples/st_things/light/sample_light_user.c

```diff
@@ -34,6 +34,7 @@ static const char *PROPERTY_VALUE_COLOR_TEMPERATURE = "ct";
 
 static char *power_status[] = { "off", "on" };
```

Comment:
Could you please match rule of variable naming also.
If it is global variable, begins 'g_'.
ex) g_power_status

Reply:
fixed.